src/index.js

```diff
@@ -1,17 +1,5 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import ReactDom from 'react-dom';
+import TodoContainer from './components/TodoContainer.js';
```

Comment:
- [ ] The imported `./components/TodoContainer` file is named `/components/toDOContainer.js`. When I fixed this error, the app worked fine.

Reply:
Thank you for the review